Makefile

```diff
@@ -86,7 +86,7 @@ install-app-selinux:
 install-app-config:
 	install -d $(AGENTDDIR)
 	$(foreach app,$(APPS), \
-	    install -p -m 644 app/$(app)/userparameters/$(app).conf $(AGENTDDIR) || :; \
+	    install -p -m 644 app/$(app)/userparameters/*$(app).conf $(AGENTDDIR) || :; \
```

Comment:
This was done according to our Jabber discussion, were you preferred to have wildcards instead of a fixed prefix...